docs/compilers/CSharp/Compiler Breaking Changes - DotNet 8.md

```diff
@@ -0,0 +1,28 @@
+# This document lists known breaking changes in Roslyn after .NET 7 all the way to .NET 8.
+
+## Inferred delegate type for methods with default parameter values or `params` arrays is no longer `System.Action` or `System.Func`
```

Comment:
>Inferred delegate type for methods with default parameter values or `params` arrays is no longer `System.Action` or `System.Func` [](http://example.com/codeflow?start=3&length=129)

Should the breaking change be generalized to "Inferred delegate type for methods includes default parameter values and `params`", since `Action` and `Func` were only used for otherwise compatible signatures previously?